worker/jobTypes/githubJob.js

```diff
@@ -40,7 +41,16 @@ class GitHubJobClass {
         }
         return false;
     }
-
+    async constructManifestIndexPath(){
+        try {
+            const snootyName = await utils.getSnootyProjectName(this.getRepoDirName());
+            this.currentJob.payload.manifestPrefix = snootyName + '-' + (this.currentJob.payload.alias ? this.currentJob.payload.alias : this.currentJob.payload.branchName)
+        } catch (error) {
+            console.log(error)
```

Comment:
`console.error()` is preferable to `console.log()` for errors, unless it actually needs to be on stdout

Reply:
Actually, maybe `console.trace()` would be appropriate here?

Reply:
Hmm we use console.log error in all of our try/catch blocks (probably not a good thing?). I see one instance of console.error already in the repo, so I'll use console.error here I think.

Reply:
On second thought - I think we should use our logging class for this!

Reply:
(not marking this as resolved just so you can give thumbs up or down)